tests/strictEqual.test.js

```diff
@@ -0,0 +1,82 @@
+const describe = require('mocha').describe
+const it  = require('mocha').it
+const expect = require('chai').expect
+const strictEqual = require('../strictEqual')
+
+describe('strictEqual', () => { 
+    describe('strictEqual', () => { 
+        it('returns true if values provided are same and are of the same type',() => {
+            // create test data
+
+            const valueOne = '5' 
+            const valueTwo = '5'
+
+
+
+            // call the function with the test data
+
+            const Return = strictEqual(valueOne,valueTwo)
+
+            //make assertions about what should be true
+
+            expect(Return).to.equal(true) 
+        })
+        it('returns false if values provided are different and are of  the same type',() => {
+            // create test data
+
+            const valueOne = 'a'
+            const valueTwo = '5' 
+
+
+
+            // call the function with the test data
+
+            const Return = strictEqual(valueOne,valueTwo)
+
+            //make assertions about what should be true
+
+            expect(Return).to.equal(false) 
+        })
+        it('returns false if values provided are same and are of diffrent type',() => {
+            // create test data
+
+            const valueOne = 5  
+            const valueTwo = '5'
+
+
+
+            // call the function with the test data
+
+            const Return = strictEqual(valueOne,valueTwo) 
```

Comment:
Since this data is simplistic, you could one line this. Also, don't use keywords for variable names. `return` is a JavaScript word with meaning so we avoid it as a name. I'm betting that's the reason you went with `Return` but remember with camelCase the first word is always all lower case and we only upper case the first letter of each successive word.

```JavaScript
const result = strictEqual(5, '5')
```